cypress.json

```diff
@@ -0,0 +1,6 @@
+{
+    "baseUrl": "http://localhost:8000",
+    "env": {
+        "login_url": "/login"
```

Comment:
Should you be using this in the tests as `Cypress.env()`?

Reply:
Yes, yes I should. The way I'm calling it right now just appends the string to the `baseUrl`. I'll change it to use `Cypress.env()`